modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm

```diff
@@ -248,3 +248,22 @@
 	new /obj/item/grenade/spawnergrenade/manhacks(src)
 	for(var/i in 1 to 6)
 		new /obj/item/pen/edagger(src)
+
+/obj/item/storage/box/syndie_kit/loadout/greytide/PopulateContents() //hope you like reading baby, this kit's all about little initial power for massive endgame potential- you have staying power with energy weapons and other armaments but need to find or make them yourself, so GET TIDING
+	new /obj/item/spear/grey_tide(src)
```

Comment:
I feel like this could be just a plasmaglass spear rather than this fancy spear that's admin-only.

Reply:
Well, the spear is the kit's sole novelty item, in total fairness. The vibroblade was in a similar admin-only state until also granted to the Cyborg Ninja loadout. It's not as powerful as one might imagine- I've run the tests and the like, and it mostly shines in group combat rather than being focused down by three seccies.

Reply:
It's still a flat 50 percent chance on hit to create a duplicate, from what I've noticed, which is still kinda strong. If it's the main piece of the whole thing, then it should be the main focus too.

Reply:
It is strong, but in testing it's unable to beat more than a lone security officer, and for the high price of 30 telecrystals I think it's fair. If I removed it, I'd lower it to 20 or less.

Reply:
maybe make it a subtype of the plasma spear that looks like the admin one?

Reply:
thing about the plasma spear is that it's not a subtype, it's a variable difference in icon and the only change is that it has slightly higher thrown force (23)